.github/workflows/lint-test.yaml

```diff
@@ -70,3 +71,6 @@ jobs:
 
       - name: Run chart-testing (install)
         run: ct install --config .github/config/ct.yaml
+
+      - name: Run chart-testing for kserve (install)
```

Comment:
will this run for every build? or only when chart was changed?

Reply:
It should run only when the chart was change, `ct` will compare to master branch by default. @maniteja6799 pls cmiiw